wamr-compiler/main.c

```diff
@@ -68,6 +68,13 @@ print_help()
     printf("  --xip                     A shorthand of --enalbe-indirect-mode --disable-llvm-intrinsics\n");
     printf("  --enable-indirect-mode    Enalbe call function through symbol table but not direct call\n");
     printf("  --disable-llvm-intrinsics Disable the LLVM built-in intrinsics\n");
+    printf("  --enable-builtin-intrinsics=<i64.common,f64.common>\n");
+    printf("                            Enable the specified built-in intrinsics, it will override the default\n");
+    printf("                              settings.\n");
+    printf("                            It only takes effect when --disable-llvm-intrinsics is set.\n");
```

Comment:
Sorry, I meant to merge the next two lines into one:
```C
    printf("                            Enable the specified built-in intrinsics, it will override the default\n");
    printf("                            settings. It only takes effect when --disable-llvm-intrinsics is set.\n");
```
Do you think it is better?

Reply:
OK, let's merge them